rm_reporting/controllers/party_controller.py

```diff
@@ -59,8 +64,12 @@ def get_enrolment_data(survey_id: str, business_ids: str) -> list:
 
     enrolment_details_query = text(enrolment_details_query_text)
     logger.info("About to get enrolment details", survey_id=survey_id)
-    with party_engine.begin() as conn:
-        enrolment_details_result = conn.execute(enrolment_details_query, {"survey_id": survey_id}).all()
+    try:
+        with party_engine.begin() as conn:
+            enrolment_details_result = conn.execute(enrolment_details_query, {"survey_id": survey_id}).all()
+    except SQLAlchemyError:
+        logger.error("Failed to get enrolment details", survey_id=survey_id)
```

Comment:
As above and while I'm not sure the survey_id is particularly useful here we don't want to print out all the party ids

Reply:
Done, including removing the `survey_id`.